chainer/links/connection/convolution_2d.py

```diff
@@ -51,6 +54,10 @@ class Convolution2D(link.Link):
             convolution operation will be executed independently. Input channel
             size ``in_channels`` and output channel size ``out_channels`` must
             be exactly divisible by this value.
+        standardizeW (bool): A flag to use the weight standardization from the
+            paper: `Weight Standardization <https://arxiv.org/abs/1903.10520>`_
+        eps (float): Small value for the numerical stability in
```

Comment:
The name `eps` is too broad to be used in weight standardization.

Reply:
Since re-implementing as `link_hook`, This arguments `eps` in `Convolution2D` is removed.